tests/remote/test_fennec.py

```diff
@@ -19,8 +19,8 @@
     # Support for API level 15 ended on Aug 29th 2017
     {'application': 'fennec', 'platform': 'android-api-15',
      'branch': 'mozilla-central', 'date': '2017-08-29'},
-    {'application': 'fennec', 'platform': 'android-api-16', 'branch': 'mozilla-central'},
-    {'application': 'fennec', 'platform': 'android-x86', 'branch': 'mozilla-central'},
+    {'application': 'fennec', 'platform': 'android-api-16', 'branch': 'mozilla-central','date':'2019-05-16'},
+    {'application': 'fennec', 'platform': 'android-x86', 'branch': 'mozilla-central','date':'2019-05-16'},
     {'application': 'fennec', 'platform': 'android-api-16',
      'branch': 'mozilla-central', 'date': '2017-08-30'},
```

Comment:
This test data is duplicated now. Lets remove it and only keep the most recent data as changed above.